packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -1331,9 +1333,12 @@ function updateHostRoot(current, workInProgress, renderLanes) {
   const nextState: RootState = workInProgress.memoizedState;
   const root: FiberRoot = workInProgress.stateNode;
 
+  if (enableTransitionTracing) {
+    pushRootTransition(workInProgress, root, renderLanes);
+  }
+
   if (enableCache) {
     const nextCache: Cache = nextState.cache;
-    pushRootTransition(root);
```

Comment:
We can't just deleted this line because `enableTransitionTracing` might be `false` while `enableCache` is true

Reply:
Resolved out of band